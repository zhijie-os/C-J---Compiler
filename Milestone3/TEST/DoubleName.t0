void foo(){}
int x;

main()
{

}