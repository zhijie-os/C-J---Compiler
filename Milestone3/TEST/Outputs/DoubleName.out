PROGRAM
    FUNC_DEC
        VOID
        identifier { line: 1, literal: foo, address: 0x561848144eb8 returnType = VOID paramType = <  >  }
        FORMALS
        BLOCK
    GLOBAL_VAR_DEC
        int { line: 2 }
        identifier { line: 2, literal: x, address: 0x561848144f18 type = INT }
    MAIN_DEC
        VOID
        identifier { line: 4, literal: main, address: 0x561848144fd8 returnType = VOID paramType = <  >  }
        FORMALS
        BLOCK
